src/vm/wren_debug.c

```diff
@@ -2,17 +2,31 @@
 
 #include "wren_debug.h"
 
-void wrenDebugPrintStackTrace(ObjFiber* fiber)
+void wrenDebugPrintStackTrace(WrenVM* vm, ObjFiber* fiber)
 {
   if (IS_STRING(fiber->error))
   {
-    fprintf(stderr, "%s\n", AS_CSTRING(fiber->error));
+    if ( vm->config.errorFn != NULL)
```

Comment:
Instead of checking for `NULL` every time you use this, how about just initializing it in `wrenInitConfiguration()` to a function that calls `fprintf(stderr, ...)`?